data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -40,11 +40,12 @@ spec:
             - -ce
             - |
               cni_mount_dir=/opt/cni/bin
+              host_dir={{ .CNIBinDir }}
               components=("bridge" "tuning")
               for component in "${components[@]}"; do
                 if find "${cni_mount_dir}/${component}" &>/dev/null && ! test -L "${cni_mount_dir}/${component}"; then
                   echo "${component} binary found, creating symbolic link ${cni_mount_dir}/cnv-${component}"
-                  ln -sf "${cni_mount_dir}/${component}" "${cni_mount_dir}/cnv-${component}"
+                  ln -sf "${host_dir}/${component}" "${cni_mount_dir}/cnv-${component}"
```

Comment:
Should not be the source for the link on host_dir as well?

Reply:
the source is the changed one
we can see on OCP that the soft link is created in the right folder (since this folder is mounted)
but the target is wrong as it points to the mount instead the real path
so i think it is good, make sense ?
will double check on CRC

Before fix, `/opt/cni/bin` is the mount point instead the real host path where it should point
```
$ ls -l /var/lib/cni/bin/
total 398436
-rwxr-xr-x. 1 root root  4010197 Jul 20 16:43 bandwidth
-rwxr-xr-x. 1 root root  4497776 Jul 20 16:44 bond
-rwxr-xr-x. 1 root root  4531999 Jul 20 16:43 bridge
lrwxrwxrwx. 1 root root       19 Jul 30 17:24 cnv-bridge -> /opt/cni/bin/bridge   ## binary is missing
lrwxrwxrwx. 1 root root       19 Jul 30 17:24 cnv-tuning -> /opt/cni/bin/tuning   ## binary is missing
```